doc/source/_static/css/examples.css

```diff
@@ -85,6 +85,10 @@
   line-height: 20px;
 }
 
+.tag > span {
+  white-space: pre;
```

Comment:
should use `nowrap`?

pre means preserving continuous white spaces as they are.

Reply:
Sure, that works too. We don't have any linebreaks or extra spaces, but I can change this anyway.

Reply:
and maybe also add `min-width: auto`